app/models/provider_enrichment.rb

```diff
@@ -34,7 +34,7 @@ class ProviderEnrichment < ApplicationRecord
 
   scope :latest_created_at, -> { order(created_at: :desc) }
   scope :latest_published_at, -> { order(last_published_at: :desc) }
-  scope :draft, -> { where(status: 'draft') }
+  scope :draft, -> { where(status: 'draft').or(ProviderEnrichment.rolled_over) }
```

Comment:
I think this is advised against in rails 6...

Reply:
What part @dankmitchell ?

Reply:
`DEPRECATION WARNING: Class level methods will no longer inherit scoping from `draft` in Rails 6.1.`